src/_lib/sass/src/_modals.scss

```diff
@@ -53,19 +52,11 @@
         .modal-content {
             background-color: #ffffff;
         }
-        .modal-header {
-            background-color: get-color("gray", 6x-light);
-
-            .modal-title,
-            .close {
-                color: get-color("gray", x-light);
-            }
-        }
-        .modal-body {
-            color: get-color("gray", base);
-        }
+        .modal-header,
+        .modal-title,
+        .modal-body,
         .modal-footer {
-            border-top-color: get-color("gray", 5x-light);
+            color: $level1;
```

Comment:
Setting text color to one of our levels worries me a little. Any changes to our levels would have the unexpected side effect of changing this text color. I'm not sure what the best solution to this is

Reply:
Sure, we should discuss with Colm. The inverse stuff is not even included in the designs yet so I'm improvising since Colm didn't explicitly ask me to ignore it for the moment.

Reply:
Discussed with Colm, reverting back to grays on bright backgrounds for now.